<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseConfig</class>
 <widget class="QWidget" name="DatabaseConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="kcfg_UseServer">
     <property name="toolTip">
      <string>Check to enable the use of an external database. By default there is already an embedded MySQL database running.</string>
     </property>
     <property name="whatsThis">
      <string>Check to enable the use of an external database. By default there is already an embedded MySQL database running.</string>
     </property>
     <property name="text">
      <string>Use &amp;external MySQL database</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="group_Connection">
     <property name="title">
      <string>Connection Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_Host">
          <property name="text">
           <string>&amp;Server:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>kcfg_Host</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QLineEdit" name="kcfg_Host">
          <property name="toolTip">
           <string>Enter the name of the existing MySQL server
</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_Port">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>P&amp;ort:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>2</number>
          </property>
          <property name="buddy">
           <cstring>kcfg_Port</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QSpinBox" name="kcfg_Port">
          <property name="toolTip">
           <string>Enter the port number to access the server. Default is port 3306.
</string>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_User">
          <property name="text">
           <string>&amp;Username:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>kcfg_User</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="5">
         <widget class="QLineEdit" name="kcfg_Password">
          <property name="toolTip">
           <string>Enter the password for the default user.
</string>
          </property>
          <property name="inputMask">
           <string notr="true"/>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_Password">
          <property name="text">
           <string>&amp;Password:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>kcfg_Password</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="5">
         <widget class="QLineEdit" name="kcfg_User">
          <property name="toolTip">
           <string>Enter the username to access the database with full access privileges.
</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_Database">
          <property name="text">
           <string>&amp;Database:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>kcfg_Database</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="5">
         <widget class="QLineEdit" name="kcfg_Database">
          <property name="toolTip">
           <string>Enter the name of the database. Default preset is amarokdb.
</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="5">
         <widget class="QPushButton" name="button_Test">
          <property name="text">
           <string>Test database connection</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame_info">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_Info">
           <property name="text">
            <string>Amarok expects the above database and user account to already exist. This user also requires full access to the database. Changes only take effect after Amarok is restarted.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_SQL">
           <property name="text">
            <string>If your database is not already set up, you can use the following S&amp;QL commands (after replacing the password with the correct one):</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="buddy">
            <cstring>text_SQL</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="text_SQL">
           <property name="cursor" stdset="0">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
